index.html

```diff
@@ -23,13 +23,20 @@
             <header>
                 <nav class="flex-container">
                     <div class="logo-div">My Logo</div>
-                    <div class="hamburger_icon"> <img src="img/hamburger-menu-icon.webp" width="20">
+                    <div class="hamburger_icon mobile"> <img src="img/hamburger-menu-icon.png" width="20">
+                    </div>
+                    <div class="desktop">
+                        <ul class="navlink">
+                            <li>Portfolio</li>
+                            <li>About</li>
+                            <li>Contact</li>
+                        </ul>
                     </div>
                 </nav>
             </header>
             <section class="headclass">
                 <h1 class="section_head">
-                    I’m Mariane Glad to see you!
+                    I’m Mariane <br> Glad to see you!
```

Comment:
[OPTIONAL] 👍🏽 

- I am not sure I am reviewing Mariane's Pull Request, Can we change this to KingNoel ?  🌸